x-pack/plugins/dataset_quality/server/routes/data_streams/routes.ts

```diff
@@ -135,8 +136,36 @@ const dataStreamDetailsRoute = createDatasetQualityServerRoute({
   },
 });
 
+const estimatedDataInBytesRoute = createDatasetQualityServerRoute({
+  endpoint: 'GET /internal/dataset_quality/data_streams/estimated_data',
+  params: t.partial({
+    query: t.intersection([rangeRt, typeRt]),
```

Comment:
why all the parameters are optional? at least `type` should always be present imo.
Also, we want to always get the estimatedDataInBytes for an specific timeRange isn't it?

Reply:
Did type disappeared from the params? I think we should add it again. In general the whole dataset quality is thought to not only support logs (future vision)